plugin.video.carnet-meduza/resources/lib/carnet-meduza.py

```diff
@@ -0,0 +1,270 @@
+# -*- coding: utf-8 -*-
+import os, sys
+import urllib, urlparse, json
+import xbmcgui, xbmcplugin, xbmcaddon
+import requests
+from register import device
```

Comment:
IMHO it's not good practice to have the init code execute during the import, particularly in this case where the module is possibly waiting for keyboard input during the import. This could lead to strange load order issues.  The better way to do that is add the init code to the class init, you could also pass the addon name as a parameter. I would do that from your default.py. that would encapsulate the init logic and allow the module to be used more easily elsewhere.

there's a lot of logic there to accomplish some fairly simple things. You might want to have a look at the module in the Kodi repo: https://github.com/xbmc/repo-scripts/tree/jarvis/script.module.simplecache